### ISSUE TITLE
Accessibility issue with incorrect tab order for multiple inputs.

### ISSUE THREAD

**Issue by tracytang11** [SAP/fundamental-ngx#9914]
#### Is this a bug, enhancement, or feature request?

Accessibility Issue.

##### Observed Behavior:
On accessing list box 'Buyers', it is observed that there are two tab stops present for the list box.

##### Expected Behavior: 
On accessing the UI elements using keyboards, the visible focus should follow the exact keyboard commands. Here there should be only single tab provided stop for the ‘Buyers’ list box.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)

angular - 14.2.4
fundamental ngx - 0.36.3

#### If this is a bug, please provide steps for reproducing it.

This issue:
The first tab selected input box:
<img width="1156" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/134352250/bd0789fb-1011-46dd-b5bf-1bba359cffd1">

The second tab selected the addon button:
![image](https://github.com/SAP/fundamental-ngx/assets/134352250/31026729-5503-43a8-8862-0999fd9b4cae)

##### How to reproduce: 
1. Open this link: [Fundamental Test](https://stackblitz.com/run?file=src%2Fmain.ts)
2. Use the tab key multiple times to focus on the multi-input box.

#### Is there anything else we should know?
About the Fundamental library component multi-input , the logic of keyboard control multi-input is up to you.
So please help us improve this.
Thanks!

**Comment by droshev** [comment]
@nikolay-kolarov What is your insight on this ticket/request?
